tests/test_mirror_port_erspan.py

```diff
@@ -0,0 +1,550 @@
+
+        """
+        dvs.setup_db()
+        pmap = dvs.counters_db.get_entry("COUNTERS_PORT_NAME_MAP", "")
+        pmap = dict(pmap)
+
+        session = "TEST_SESSION"
+        src_ports = "Ethernet0,Ethernet4"
+        src_asic_ports = ["Ethernet0", "Ethernet4"]
+
+        # create mirror session
+        self.dvs_mirror.create_erspan_session(session, "10.10.10.10", "11.11.11.11", "0x6558",
+                                              "8", "100", "0", None, src_ports, "TX")
+        self.dvs_mirror.verify_session_status(session, status="inactive")
+
+        # create port channel; create port channel member
+        self.dvs_lag.create_port_channel("008")
+        lag_entries = self.dvs_vlan.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG", 1)
+        self.dvs_lag.create_port_channel_member("008", "Ethernet88")
+                                                
+        # Verify the LAG has been initialized properly
+        lag_member_entries = self.dvs_vlan.asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", 1)
+        fvs = self.dvs_vlan.asic_db.wait_for_entry("ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER", lag_member_entries[0])
+        assert len(fvs) == 4
+        assert fvs.get("SAI_LAG_MEMBER_ATTR_LAG_ID") == lag_entries[0]
+        assert self.dvs_vlan.asic_db.port_to_id_map[fvs.get("SAI_LAG_MEMBER_ATTR_PORT_ID")] == "Ethernet88"
+
+        # bring up port channel and port channel member
+        dvs.set_interface_status("PortChannel008", "up")
+        dvs.set_interface_status("Ethernet88", "up")
+
+        # add ip address to port channel 008
+        dvs.add_ip_address("PortChannel008", "11.11.11.0/24")
+        self.dvs_mirror.verify_session_status(session, status="inactive")
+
+        # create neighbor to port channel 008
+        dvs.add_neighbor("PortChannel008", "11.11.11.11", "88:88:88:88:88:88")
+        self.dvs_mirror.verify_session_status(session)
+
+        # Check src_port state.
+        expected_asic_db = {"SAI_MIRROR_SESSION_ATTR_MONITOR_PORT": pmap.get("Ethernet88"),
+                            "SAI_MIRROR_SESSION_ATTR_DST_MAC_ADDRESS": "88:88:88:88:88:88"}
+
+        self.dvs_mirror.verify_session(dvs, session, asic_db=expected_asic_db, src_ports=src_asic_ports, direction="TX")
+
+        # remove neighbor
+        dvs.remove_neighbor("PortChannel008", "11.11.11.11")
+        self.dvs_mirror.verify_session_status(session, status="inactive")
+
+        # remove ip address
+        dvs.remove_ip_address("PortChannel008", "11.11.11.0/24")
+        self.dvs_mirror.verify_session_status(session, status="inactive")
+
+        # bring down port channel and port channel member
+        dvs.set_interface_status("PortChannel008", "down")
+        dvs.set_interface_status("Ethernet88", "down")
+
+        # remove port channel member; remove port channel
+
+        self.dvs_lag.remove_port_channel_member("008", "Ethernet88")
+        self.dvs_lag.remove_port_channel("008")
```

Comment:
Same for LAGs, it's a good idea just to confirm they have been cleaned up at the end of the test case before moving on (although if there is no `verify_no_lag_*` methods like there are for VLANs at the moment I think you can leave this as is for now).

Reply:
Done. Added checks.